backend/elearning/settings.py

```diff
@@ -1,7 +1,7 @@
-import environ
-
 from pathlib import Path
 
+import environ
+
 env = environ.Env()
```

Comment:
be careful when rebasing next time, since we may get some unnecessary changes like this

Reply:
moved by linter/isort